src/types.d.ts

```diff
@@ -243,6 +243,7 @@ interface Destruction {
   finalisedAt?: string
   finalisedBy?: User['id']
```

Comment:
@salony75way for the two new `reportPrinted` fields, could you rename them to `reportPrintedAt` - to match other date fields please?